.github/workflows/publish_release.yml

```diff
@@ -1,9 +1,12 @@
 name: Publish release
+
```

Comment:
Please remove this newline change and the one below. It's not really related to the patch itself and the other workflows also have no newlines here. (If we want that, it should be done consistently in all workflows in a separate PR.)

Reply:
Fixed, thanks!